js/skin.js

```diff
@@ -706,7 +706,7 @@ const Skin = createReactClass({
       }
     }
 
-    var className = ClassNames('oo-responsive', {
+    var className = ClassNames(this.state.responsiveClass, 'oo-responsive', {
```

Comment:
Where do you set value for responsiveClass state?

Reply:
@KseniiaSokolovaEpam that value comes from `ResponsiveManagerMixin` used as a mixin at the Skin component at line 32